source/excerptexport/templates/excerptexport/create_excerpt_export.html

```diff
@@ -1,4 +1,4 @@
-{% extends "excerptexport/layouts/base.html" %}
```

Comment:
What's the reason for flatting the template structure? We used layouts, templates and partials to structure the different types of templates.